crates/router/src/connector/adyen/transformers.rs

```diff
@@ -213,6 +218,25 @@ pub enum AdyenPaymentMethod {
     AfterPay(AdyenPayLaterData),
     AdyenKlarna(AdyenPayLaterData),
     AdyenAffirm(AdyenPayLaterData),
+    Eps(BankRedirectionWithIssuer),
+    Ideal(BankRedirectionWithIssuer),
+    Giropay(BankRedirectionPMData),
+    Sofort(BankRedirectionPMData),
+}
+
+#[derive(Debug, Clone, Serialize, Deserialize)]
```

Comment:
Do we need both `serialize` and `deserialize` over these structs
Also see to all the structs that have been implemented, remove wherever redundant